integration/obscuroscan/obscuroscan_test.go

```diff
@@ -65,12 +64,19 @@ func TestObscuroscan(t *testing.T) {
 	require.NoError(t, err)
 
 	// wait for the msg bus contract to be deployed
-	time.Sleep(10 * time.Second)
+	time.Sleep(30 * time.Second)
 
 	// make sure the server is ready to receive requests
 	err = waitServerIsReady(serverAddress)
 	require.NoError(t, err)
 
+	issueTransactions(
+		t,
+		fmt.Sprintf("ws://127.0.0.1:%d", startPort+integration.DefaultHostRPCWSOffset),
+		wallet.NewInMemoryWalletFromConfig("8dfb8083da6275ae3e4f41e3e8a8c19d028d32c9247e24530933782f2a05035b", integration.ObscuroChainID, testlog.Logger()),
```

Comment:
Maybe add a comment what `dfb8083da6275ae3e4f41e3e8a8c19d028d32c9247e24530933782f2a05035b` means.

Reply:
Linked it to the genesis 👍